sunpy/net/dataretriever/sources/tests/test_fermi_gbm.py

```diff
@@ -93,7 +93,7 @@ def test_client_repr(LCClient):
     Repr check
     """
     output = str(LCClient)
-    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClien'
+    assert output[:50] == 'sunpy.net.dataretriever.sources.fermi_gbm.GBMClient'
```

Comment:
interesting choice tbh to do 50 and not 51

Reply:
I did this as I was lazy and like the number 50.